DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/SRSnapshotTests.swift

```diff
@@ -7,6 +7,7 @@
 import XCTest
 import SRFixtures
 import TestUtilities
+import DatadogInternal
```

Comment:
question/ Why we need `import DatadogInternal` in this scope?

Reply:
Good point. I needed it because of this line: `let privacyModes: [SessionReplayPrivacyLevel] = [.allow, .mask]`. It was convenient during tests, but I can remove it now.

Reply:
Oh, this is actually a bug 🐞 that will require our customers to also import `DatadogInternal`, which we rather avoid. To fix it, the `SessionReplayPrivacyLevel` must be re-exported from `DatadogInternal` in [`SessionReplayConfiguration.swift`](https://github.com/DataDog/dd-sdk-ios/blob/develop/DatadogSessionReplay/Sources/SessionReplayConfiguration.swift). The same way as we do for some types in RUM: [example](https://github.com/DataDog/dd-sdk-ios/blob/380bdf4938519b676d5a1d06bd2ce6928637a0a6/DatadogRUM/Sources/RUMConfiguration.swift#L10-L17). // cc @maxep 

For simplicity, can we fix it within this PR so we get rid of this import in snapshot tests?

Reply:
I already removed the import statement from this file, but I could still do this requested change in the follow-up, refactoring PR?

Reply:
Ah yes, that's a miss from me 🤦  I can open another PR to fix it 👍